File: programmers/level1/두 개 뽑아서 더하기.swift
import Foundation

func solution(_ numbers:[Int]) -> [Int] {
    var resultSet = Set<Int>()
    for i in 0..<(numbers.count - 1) {
        for j in (i + 1)..<numbers.count {
            resultSet.insert(numbers[i] + numbers[j])
        }
    }
    return Array(resultSet).sorted()
}

print(solution([2, 1, 3, 4, 1]))

File: programmers/level1/두개뽑아서더하기.py
def solution(numbers):
    answer = []
    for i in range(len(numbers) - 1):
        for j in range(i + 1, len(numbers)):
            answer.append(numbers[i] + numbers[j])
    answer = set(answer)
    return sorted(answer)


numbers = [2, 1, 3, 4, 1]

print(solution(numbers))

